website/docs/reference/project-default-strategy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Project default strategy
+---
+
+:::info Availability
+
+The project default strategy feature is generally available starting with **Unleash 5.2.0**.
+
+:::
+
+A project's environment's **default strategy** specifies **the activation strategy that will be used when you enable a feature in that environment, if the environment has no defined strategies)**. 
+
+## Limitations
+
+The default project strategy uses the [Gradual Rollout activation strategy](/reference/activation-strategies.md) as it's base.  You can customise to your needs by using constraints or segments (enterprise only)
```

Comment:
This header doesn't seem to apply to the text, so I think we can just remove it. (Plus some minor text suggestions and links)

```suggestion
All default project strategies use the [gradual rollout activation strategy](activation-strategies.md). By default, the rollout 100%. You can customize the strategies by changing the rollout percentage and adding [constraints](strategy-constraints.mdx) and [segments](segments.mdx) as you would for any other strategy.
```